src/lib/types/Product.ts

```diff
@@ -95,6 +95,7 @@ export const MAX_DESCRIPTION_LIMIT = 10000;
 
 export const DEFAULT_MAX_QUANTITY_PER_ORDER = 10;
 export const POS_PRODUCT_PAGINATION = 10;
+export const PRODUCT_PAGINATION_LIMIT = 15;
```

Comment:
```suggestion
export const PRODUCT_PAGINATION_LIMIT = 25;
```

as discussed in the issue